Searching & sorting/shell_sort.py

```diff
@@ -0,0 +1,16 @@
+def shellSort(input_list):
+   gap = len(input_list) // 2
+   while gap > 0:
+      for i in range(gap, len(input_list)):
```

Comment:
add a comment here for understanding

Reply:
try using different variable not inbuilt variable '"list"  as "arr" or something else